js/register.js

```diff
@@ -25,11 +30,27 @@ Register.prototype.render = function() {
 	'</form>';
 
   // insert content into `el`
+  this.el.html(templateString);
   // add any event listeners
+  var entry = this.el.find('.entry');
+  entry.on('submit', this.update.bind(this));
 };
 
-Register.prototype.update = function() {
+Register.prototype.update = function(event) {
   event.preventDefault();
-  var inputBox = this.el.find($(".newEntry"));
-  
+  var inputBox = this.el.find('.newEntry');
+  var entries = this.el.find('.entries');
+  var total = this.el.find('.total');
+
+  var userInput = parseFloat(inputBox.val());
+  inputBox.val('');
+
+  var newEntry = $('<tr> <td></td> </td>');
+  newEntry.append('<td>$' + userInput.toFixed(2) + '</td>');
+
+  entries.append(newEntry);
+
+  this.total += userInput;
+  total.text('$' + this.total.toFixed(2));
```

Comment:
line 53: this.total is the jQuery object returned on line 43, and userInput is a number. It doesn't make sense to add the two together.